### PR TITLE
PE-2920: fixes get last modified date in firefox

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
--- Releases ---
iOS release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:ios:06ea33a95a2e0d42ffce07/releases/25nspupbvk7s8
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/6tthg25hk89ro

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -379,11 +379,11 @@ packages:
     source: hosted
     version: "6.1.2"
   file_picker:
-    dependency: "direct overridden"
```
Comment (matibat) [inline pubspec.lock:382]: So now `file_picker` depends on `ardrive_io`?

File: pubspec.yaml [hunk 2]
```diff
@@ -35,7 +35,7 @@ dependencies:
   ardrive_io:
     git:
       url: https://github.com/ar-io/ardrive_io.git
-      ref: 9dc9806993ff9a35d72b88460ac5c91f3ed33c81
+      ref: PE-2920
```
Comment (matibat) [inline pubspec.yaml:38]: Reminder to BUMP this when released

**Review by matibat — APPROVED** [review]
LGTM

File: pubspec.lock (hunk 1 above)
Comment (thiagocarvalhodev) [reply pubspec.lock:382]: Yep!! We're overriding before because I didn't have access to push the new version to the `ardrive_io` to change there. 🤓

**Pull request merged**